### ISSUE TITLE
Expo CLI won't init Bare Worklfow Package

### ISSUE THREAD

**Issue by siddharthsharma94** [expo/expo-cli#1536]
# Required format for issues

Please make our job easier by filling this template out to completion.

## Description

1-2 sentences describing the problem you're having or the feature you'd like to request

## Expected Behavior
Cannot init  expo bare workflow
What action did you perform, and what did you expect to happen?
```
➜  ~ expo init --template bare-minimum
? What is the name of your project? newAppTest
Using Yarn to install packages. You can pass --npm to use npm instead.
The "path" argument must be of type string. Received type undefined
Set EXPO_DEBUG=true in your env to view the stack trace.
➜  ~ 
```

## Observed Behavior
It should make a new project, and it works with managed workflow but not bare.
What actually happened when you performed the above actions?

If there's an error message, please paste the _full terminal output and error message_ in this code block:

```
The "path" argument must be of type string. Received type undefined
```

## Environment

Please run `expo diagnostics` in the project folder and copy the output here:
```

  Expo CLI 3.11.9 environment info:
    System:
      OS: macOS 10.15.3
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 10.16.3 - /usr/local/bin/node
      Yarn: 1.21.1 - ~/.yarn/bin/yarn
      npm: 6.13.4 - ~/.npm-packages/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.5 AI-191.8026.42.35.5900203
      Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
    npmGlobalPackages:
      expo-cli: 3.11.9
```

**Comment by brentvatne** [comment]
cc @fson this may have been caused by recent change to simplify init

@siddharthsharma94 - for now you can just run `expo init` without passing in the flag and then pick the template and it will work

**Comment by siddharthsharma94** [comment]
I tried that, and after selecting the bare bones workflow (with and without typescript) it still doesn’t seem to work.

**Comment by brentvatne** [comment]
odd, it works as expected on my machine using the latest version of expo-cli...

```
╭─~/coding
╰─$ expo init testing
? Choose a template: expo-template-bare-typescript
Using Yarn to install packages. You can pass --npm to use npm instead.
Extracting project files...
Customizing project...
Initialized a git repository.
Installing dependencies...
yarn install v1.19.2
info No lockfile found.
[1/4] 🔍  Resolving packages...
```

**Comment by siddharthsharma94** [comment]
And that worked! Thanks!

**Comment by brentvatne** [comment]
just published expo-cli@3.12.0 with this fix. i'll set it as the latest tag tomorrow, but for now you can install it with `npm i -g expo-cli@3.12.0`
